WordPress/Classes/ViewRelated/Gutenberg/GutenbergViewController.swift

```diff
@@ -875,7 +875,13 @@ extension GutenbergViewController: GutenbergBridgeDataSource {
     }
 
     private var isUnsupportedBlockEditorEnabled: Bool {
-        return !(post.blog.jetpack?.isConnected ?? false)
+        // The Unsupported Block Editor is made for
```

Comment:
Can we change that comment and explain which kind of sites are not supported and why? 

From what I understood, only Jetpack sites that are not Atomic won't work because there is no way to authenticate them.

Reply:
Hey @maxme - thanks for the feedback. I've updated the comment (and removed the Automated Transfer flag check as was discussed offline).